generate.py

```diff
@@ -91,10 +101,44 @@ def generate_message(media, target, conf, result):
         raise RuntimeError('Invalid media {} for {}'.format(media, target))
 
 
+def count_tweets_in_twitter_url(url):
+    useragent = ["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36","Mozilla/5.0 (Windows NT 10.0; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0","Mozilla/5.0 (Windows NT 6.1; WOW64; rv:49.0) Gecko/20100101 Firefox/49.0","Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12) AppleWebKit/602.1.50 (KHTML, like Gecko) Version/10.0 Safari/602.1.50"]
+    headers = {'User-Agent': random.choice(useragent)}
+    r = requests.get(url, headers=headers)
+    print(url,r.text.count('js-tweet-text-container'))
+    return r.text.count('js-tweet-text-container')
+
+def get_tweets(handle):
+    # get_tweets returns the number of tweets that are have tweeted to the handle about ipv6
+    # for the last NUMBER_OF_YEARS_TO_GET_TWEETS years.
+    # i.e. get_tweets("facebook") returns 133 (01/01/2013 to 10/26/2016)
+    now = datetime.datetime.now()
+    dates_to_try = []
+    for year in range(now.year - NUMBER_OF_YEARS_TO_GET_TWEETS, now.year):
+        for month in range(1, 12):
+            dates_to_try.append(str(year) + "-" + str(month))
+    for month in range(1, now.month + 1):
+        dates_to_try.append(str(now.year) + "-" + str(month))
+    # Go through tweets, one month at a time, since Twitter requires loading pages if there are too many at once
+    # (if there are too many, you may need to go one week/day at a time)
+    urls = []
+    for date_to_try in dates_to_try:
+        urls.append("https://twitter.com/search?f=tweets&q=ipv6%20%23" + handle + \
+            "%20since%3A" + date_to_try + "-01%20until%3A" + date_to_try + "-31")
+       
+    p = multiprocessing.Pool(multiprocessing.cpu_count())
```

Comment:
Again, not sure if that would be required at all if we'd be using the API.

We also are already using asyncio you may as well use something as shown in: http://stackoverflow.com/a/22414756